### PR TITLE
Irrelevant grammar fixes

### PR DESCRIPTION
**Pull request by Syncro9**
...because I wanted to contribute in any way I could. And not knowing how to Ruby doesn't help one bit.

### CODE REVIEW

File: app/views/static/about.html.haml [hunk 1]
```diff
@@ -83,7 +83,7 @@
         .panel-heading
           %h3.panel-title Funding
         .panel-body
-          %p Currently Retrospring is running of the developers funds and will be able to sustain at the current needs of servers. Hereby we want to thank the people who previously donated to our project and helped it grow. Everyone shown at the bottom of this panel has contributed to the project with donations or great input.
```
Comment (TheHowl) [inline app/views/static/about.html.haml]: Probably they meant "off"

Comment (TheHowl) [inline app/views/static/about.html.haml]: >Hereby we want

Seems fine to me

File: app/views/static/faq.html.haml [hunk 2]
```diff
@@ -9,7 +9,7 @@
...
         .panel-body
-          Retrospring was founded by nilsding and pixeldesu, initially starting as a single-user PHP script, getting rewritten in Python and reaching its current iteration written in Ruby on Rails. It was made with the purpose of bringing back social aspects to answering questions with adding another layer to the entire process, which are comments. A lot of the planned features are based off of Formspring (now known as Spring), which is now replaced by ask.fm, which we dislike for selling of our (and your data) and the censorship.
+          Retrospring was founded by Nilsding and Pixeldesu, initially starting as a single-user PHP script, thereafter getting rewritten in Python and reaching its current iteration written in Ruby on Rails. It was made with the purpose of bringing back social aspects to answering questions with adding another layer to the entire process, which are comments. A lot of the planned features are based off of Formspring (now known as Spring), which is now replaced by ask.fm, which we dislike for selling of our (and your data) and the censorship.
```
Comment (TheHowl) [inline app/views/static/faq.html.haml]: Why don't you keep the correct capitalization

Comment (nilsding) [inline app/views/static/faq.html.haml]: \>Nilsding
\>Pixeldesu

AUGH!! NO!! [#wontmerge](http://wontmerge.com/assets/img/stormtrooper.gif)

File: app/views/static/privacy_policy.html.haml [hunk 3]
```diff
@@ -2,7 +2,7 @@
   .panel.panel-default
     .panel-body
       %h1 Privacy Policy
-      %p This Privacy Policy describes how and when Retrospring uses and collects your data, checks your information and shares with other Services.
+      %p This Privacy Policy describes how and when Retrospring uses and collects your data, checks your information and shares with other Services. Herein "Service" may refer to Retrospring or anything even remotely related to it.
```
Comment (TheHowl) [inline app/views/static/privacy_policy.html.haml]: wow much legalese pro

File: app/views/static/privacy_policy.html.haml [hunk 4]
```diff
@@ -10,7 +10,7 @@
...
+      %p Like many websites, we use cookies and similar technologies to collect  additional website usage data and to improve our Services. A cookie is a small data file that is transferred to your computer's hard disk. Retrospring uses persistent cookies to improve the users browsing experience and to better understand how you interact with our Services. Most Internet browsers automatically accept cookies. You can instruct your browser, by changing it's settings, to stop accepting cookies or to prompt you before accepting a cookie from the websites you visit. However, the Services may not function properly if you disable cookies.
```
Comment (TheHowl) [inline app/views/static/privacy_policy.html.haml]: >by changing it's settings,

it's (it is) /=/ its (possessive)

File: app/views/static/terms.html.haml [hunk 5]
```diff
@@ -2,7 +2,7 @@
   .panel.panel-default
     .panel-body
       %h1 Terms of Use
-      %p These Terms of Service ("Terms") as declared by the company (referred to as "Staff") govern your access to and use of the services (the "Services" or “Retrospring”), and any  information, text, graphics, photos or other materials uploaded,  downloaded or appearing on the Services (collectively referred to as "Content").  Your access to and use of the Services are conditioned on your  acceptance of and compliance with these Terms. By accessing or using the  Services you agree to be bound by these Terms.
+      %p These Terms of Service ("Terms") as declared by the company (referred to as "Staff"), govern your access to and use of the services (the "Services" or “Retrospring”), and any information, text, graphics, photos or other materials uploaded, downloaded or appearing on the Services (collectively referred to as "Content"). Your access to and use of the Services are conditioned on your acceptance of and compliance with these Terms. By accessing or using the Services you agree to be bound by these Terms.
```
Comment (TheHowl) [inline app/views/static/terms.html.haml]: Why a comma between the subject and the verb?

File: app/views/static/terms.html.haml [hunk 6]
```diff
@@ -13,11 +13,11 @@
...
+      %p We do not endorse, support, represent or guarantee the completeness,  truthfulness, accuracy, or reliability of any Content or communications  posted via the Services or endorse any opinions expressed via the  Services excluding official information represented by the "Staff" of the Services. You understand that by using the Services, you may be exposed to Content that might be offensive, harmful, inaccurate or otherwise  inappropriate, or in some cases, postings that have been mislabelled or  are otherwise deceptive. Under no circumstances will Retrospring be liable  in any way for any Content, including, but not limited to, any errors or  omissions in any Content, or any loss or damage of any kind incurred as  a result of the use of any Content posted, emailed, transmitted or  otherwise made available via the Services or broadcast elsewhere.
```
Comment (TheHowl) [inline app/views/static/terms.html.haml]: a double space escaped here.

Comment (TheHowl) [inline app/views/static/terms.html.haml]: Actually, two of them!

File: app/views/user/_modal_privileges.html.haml [hunk 7]
```diff
@@ -12,12 +12,12 @@
           privileges
       %ul.list-group.groups--list
         - unless @user.admin?
-          = render 'user/modal_privileges_item', privilege: 'banned', description: 'Hit the user with ye olde banhammer', user: @user
+          = render 'user/modal_privileges_item', privilege: 'banned', description: 'Hit the user with ye old ban-hammer.', user: @user
```
Comment (nilsding) [inline app/views/user/_modal_privileges.html.haml]: This one was intended.

File: app/views/static/terms.html.haml (hunk 6 above)
Comment (TheHowl) [inline app/views/static/terms.html.haml]: Sorry, seven.

File: app/views/static/terms.html.haml [hunk 8]
```diff
@@ -13,11 +13,11 @@
...
+      %p We reserve the right at all times (but will not have an obligation) to remove or refuse to distribute any Content on the Services, to suspend  or terminate users, and to reclaim user names without liability to you.   We also reserve the right to access, read, preserve, and disclose any  information as we reasonably believe is necessary to satisfy any  applicable law, regulation, legal process or governmental request, enforce the Terms, including investigation of potential violations  hereof, detect, prevent, or otherwise address fraud, security or  technical issues, respond to user support requests, or protect  the rights, property or safety of Retrospring, its users and the public.
```
Comment (TheHowl) [inline app/views/static/terms.html.haml]: There are plenty of double spaces that escaped to your sight here, too! :P

File: app/views/user/_modal_privileges.html.haml (hunk 7 above)
Comment (TheHowl) [inline app/views/user/_modal_privileges.html.haml]: 1. this is a joke, it's intended for it to be olde. olde is part of ancient english terms, which explains the use of ye as a substitute of "the". so unnecessary change, except for the period.

File: app/views/static/terms.html.haml (hunk 8 above)
Comment (nilsding) [inline app/views/static/terms.html.haml]: double spaces after periods are okay though.  They increase readability on the source text imho.

**Comment by Syncro9** [comment]
Updated, I'll see if I can find more in the meantime.

File: app/views/devise/mailer/unlock_instructions.html.haml [hunk 9]
```diff
@@ -1,4 +1,4 @@
 %p Hello #{@resource.screen_name}!
-%p Your account has been locked due to an excessive number of unsuccessful sign in attempts.
-%p Click the link below to unlock your account:
+%p Your account has been locked due to excessive amount of unsuccessful sign in attempts.
```
Comment (yukimono) [inline app/views/devise/mailer/unlock_instructions.html.haml]: still needs `an excessive amount`

File: app/views/user/_modal_privileges.html.haml [hunk 10]
```diff
@@ -12,12 +12,12 @@
           privileges
       %ul.list-group.groups--list
         - unless @user.admin?
-          = render 'user/modal_privileges_item', privilege: 'banned', description: 'Hit the user with ye olde banhammer', user: @user
+          = render 'user/modal_privileges_item', privilege: 'banned', description: 'Hit the user with ye olde banhammer.', user: @user
```
Comment (yukimono) [inline app/views/user/_modal_privileges.html.haml]: This line is slated to be removed with a pull request, keep in mind if #18 gets merged into the master branch you will have to solve the merge conflicts (delete line 14 and 15).

**Comment by Syncro9** [comment]
Fixed more thingies c:

**Comment by yukimono** [comment]
hah
clever

**Comment by Syncro9** [comment]
This is actually turning out to be immensely educational for me as well. I may as well just continue doing this for a while longer.

File: app/views/devise/mailer/unlock_instructions.html.haml [hunk 11]
```diff
@@ -1,4 +1,4 @@
 %p Hello #{@resource.screen_name}!
-%p Your account has been locked due to an excessive number of unsuccessful sign in attempts.
-%p Click the link below to unlock your account:
+%p Your account has been locked due to a plethora of unsuccessful sign in attempts.
```
Comment (TheHowl) [inline app/views/devise/mailer/unlock_instructions.html.haml]: most people won't understand this imho. it's some kind of high-level english lexic

File: app/views/devise/mailer/unlock_instructions.html.haml (hunk 9 above)
Comment (Syncro9) [inline app/views/devise/mailer/unlock_instructions.html.haml]: I don't care.

File: app/views/static/about.html.haml [hunk 12]
```diff
@@ -47,7 +47,7 @@
             = APP_CONFIG['site_name']
             to get the features everyone wanted by forking our
             = succeed '.' do
-              = link_to "GitHub repository", 'https://github.com/retrospring/retrospring'
+              = link_to "GitHub repository.", 'https://github.com/retrospring/retrospring'
```
Comment (TheHowl) [inline app/views/static/about.html.haml]: the `= succeed '.'` is already there!

Comment (Syncro9) [inline app/views/static/about.html.haml]: Didn't notice sorry.

File: app/views/static/privacy_policy.html.haml [hunk 13]
```diff
@@ -2,26 +2,26 @@
...
       %p You can also permanently delete your Retrospring account with a button at the bottom of the account settings page. Removing your account will cease your ability to log in with your credentials and the loss of all your data you supplied to the Service.
       %h2 Our Policy Towards Children
-      %p Our Services are not directed to persons under 14. If you become aware that your child has provided us with personal information without your consent, contact us on privacy@retrospring.net . We do not knowingly collect personal information from children under 14. If we become aware that a child under 14 has provided us with personal information, we take steps to remove such information and terminate the child's account.
+      %p Our Services are not directed towards children under 14. If you become aware that your child has provided us with personal information without your consent, contact us on privacy@retrospring.net. We do not knowingly collect personal information from children under 14. If we become aware that a child under 14 has provided us with personal information, we take steps to remove such information and terminate the child's account.
```
Comment (TheHowl) [inline app/views/static/privacy_policy.html.haml]: Here it would be good to pit a mailto: at the email address and write explicitly `contact us via electronic email at the email address ...`

File: app/views/user/_modal_privileges.html.haml [hunk 14]
```diff
@@ -11,13 +11,11 @@
             = @user.screen_name
           privileges
       %ul.list-group.groups--list
-        - unless @user.admin?
-          = render 'user/modal_privileges_item', privilege: 'banned', description: 'Hit the user with ye olde banhammer', user: @user
```
Comment (TheHowl) [inline app/views/user/_modal_privileges.html.haml]: why did you delete these lines?

File: app/views/static/privacy_policy.html.haml (hunk 13 above)
Comment (nilsding) [inline app/views/static/privacy_policy.html.haml]: Also, please try not to make meaningless commit messages.

**Comment by Syncro9** [comment]
I think I messed up everything at this point, to be honest.

File: app/views/devise/mailer/reset_password_instructions.html.haml [hunk 15]
```diff
@@ -2,4 +2,4 @@
 %p Someone (hopefully you) has requested a link to change your password. You can do this through the link below.
 %p= link_to 'Change my password', edit_password_url(@resource, reset_password_token: @token, protocol: (APP_CONFIG['https'] ? :https : :http))
 %p If you didn't request this, please ignore this email.
-%p Your password won't change until you access the link above and create a new one.
\ No newline at end of file
+%p Your password won't change until you access the link above and insert a new one.
```
Comment (nilsding) [inline app/views/devise/mailer/reset_password_instructions.html.haml]: I think "create" fits better than "insert" bere.  A few friends of mine even suggested "set".

Comment (Syncro9) [inline app/views/devise/mailer/reset_password_instructions.html.haml]: Yeah "set" is probably the correct word.

**Comment by Syncro9** [comment]
And these are all. There may be minor mistakes which went past my eyes, do comment if you happen to find those.

File: app/views/devise/mailer/unlock_instructions.html.haml (hunk 11 above)
Comment (Syncro9) [inline app/views/devise/mailer/unlock_instructions.html.haml]: I could use "a profuse amount" or "deluge" or maybe even "overabundance".

Comment (TheCoward) [inline app/views/devise/mailer/unlock_instructions.html.haml]: Profuse amount. All the others also imply the fact that it's a good thing,
a providential plethora.

File: app/views/inbox/_entry.html.haml [hunk 16]
```diff
@@ -18,7 +18,7 @@
                 #{i.question.answer_count} response(s)
         %p.answerbox--question-text= i.question.content
   .panel-body
-    %textarea.form-control{name: 'ib-answer', placeholder: 'Write your answer here...', data: { id: i.id }}
```
Comment (nilsding) [inline app/views/inbox/_entry.html.haml]: I guess three dots are alright here UX-wise, as they imply some kind of interaction with the user.

File: app/views/moderation/_discussion.html.haml [hunk 17]
```diff
@@ -21,5 +21,5 @@
                         Delete
             %p.comments--content= comment.content
 .form-group.has-feedback{name: 'mod-comment-new-group', data: { id: report.id }}
-  %input.form-control.comments--box{type: :text, placeholder: 'Comment...', name: 'mod-comment-new', data: { id: report.id }}
+  %input.form-control.comments--box{type: :text, placeholder: 'Comment�}', name: 'mod-comment-new', data: { id: report.id }}
```
Comment (nilsding) [inline app/views/moderation/_discussion.html.haml]: "…" > "..."  ;)

File: app/views/shared/_questionbox.html.haml [hunk 18]
```diff
@@ -48,10 +48,10 @@
                 %small
                   Join
                   = APP_CONFIG['site_name']
-                  today!  You'll be able to follow and ask people you know and a lot more.
```
Comment (nilsding) [inline app/views/shared/_questionbox.html.haml]: My beloved double spaces after periods! :'(

File: app/views/static/privacy_policy.html.haml [hunk 19]
```diff
@@ -2,26 +2,30 @@
...
       %p You can also permanently delete your Retrospring account with a button at the bottom of the account settings page. Removing your account will cease your ability to log in with your credentials and the loss of all your data you supplied to the Service.
       %h2 Our Policy Towards Children
-      %p Our Services are not directed to persons under 14. If you become aware that your child has provided us with personal information without your consent, contact us on privacy@retrospring.net . We do not knowingly collect personal information from children under 14. If we become aware that a child under 14 has provided us with personal information, we take steps to remove such information and terminate the child's account.
+      %p
+        Our Services are not directed towards children under 14. If you become aware that your child has provided us with personal information without your consent, contact us on
+        = succeed '.' do
+          = link_to "on our email", 'mailto:privacy@retrospring.net'
```
Comment (nilsding) [inline app/views/static/privacy_policy.html.haml]: This will render as "[…] contact us on on our email."

Wouldn't "send us an email" or "contact us by sending an email" be better here?

File: app/views/shared/_questionbox.html.haml (hunk 18 above)
Comment (Syncro9) [inline app/views/shared/_questionbox.html.haml]: No. They aren't correct d;

Comment (nilsding) [inline app/views/shared/_questionbox.html.haml]: They improve readability in the source text (and for fixed-width fonts in general).  They aren't rendered in the browser anyway (unless you're using `links`...)

Comment (Syncro9) [inline app/views/shared/_questionbox.html.haml]: This is just about preference, really. How about we poll or something.

Comment (nilsding) [inline app/views/shared/_questionbox.html.haml]: I'm the self-appointed benevolent dictator for life when it comes to this project's sources.

I'm going to write a CONTRIBUTING.md soon-ish.

Comment (Syncro9) [inline app/views/shared/_questionbox.html.haml]: Aw shit, I'm so rekt.

**Comment by schisma** [comment]
most of these are completely useless

**Comment by Syncro9** [comment]
Which is why I said "irrelevant" in the title.

**Comment by pixeldesu** [comment]
Since they are irrelevant...

\***this pull request got hit by ye olde "pull request closed"-hammer**\*

**Comment by yukimono** [comment]
good lord

**Comment by Syncro9** [comment]
You should fix them sometime in the future though, at least the typos maybe.

I thought Retrospring was a group of perfectionists.

**Comment by pixeldesu** [comment]
We will, but most of these fixes are, in fact, irrelevant, and changing them back would be more work than just fixing the typos, thanks for pointing them out, thought!
